runtime.txt

```diff
@@ -0,0 +1 @@
+3.8
```

Comment:
It would be better to use Python 3.10 or 3.11. The maintanance period for them will be longer.

Reply:
Done.